src/grid/index.js

```diff
@@ -164,15 +179,25 @@ class Grid {
       return false;
     }).map(size => getCleanSize(size));
 
-    for (i = 0, len = sizeList.length; i < len; i++) {
+    len = sizeList.length;
+    if(tracks === "none"){
+      len = this.getProps("maxTracks");
+      console.log(len);
```

Comment:
@siddharthapaul001 This console.log statement should be removed.

@priyanjitdey94 We should probably add a lint rule to check for there errors.